index.html

```diff
@@ -332,11 +335,22 @@ <h3 class="title is-4">
                 <br>
 
                 <h3 class="title is-4">
-                    Contributing
+                    Roadmap
+                </h3>
+                <p>
+                    If you have ideas for releases in the future, it is a good idea to
+                    list them in the README.
+                </p>
+
+                <br>
+
+                <h3 class="title is-4">
+                    Contributing back
                 </h3>
                 <p>
                     If you are open to contributions from others, let them know
-                    what your requirements are for accepting contributions.
+                    what your requirements are for accepting contributions. Perhaps
+                    also how keen you are to process pull-requests, and issues.
```

Comment:
Isn't this sort of covered already by the last section regarding slowed down development?